README.md

```diff
@@ -30,6 +30,45 @@ It also has a tool to check if your data structure threw any exceptions when bei
 ;; => [{:exception e :path [:foo]}]
 ```
 
+If you have very big or even infinite lazy seqs, you are going to have a bad
+time realizing them all. By default, Realize will not accept collections with
+more than 10000 items. You can control this limit with the `:max-len` option:
+
+```clj
+(require '[realize.core :as realize])
+
+(def data (map inc (range)))
+
+(realize/realize data {:max-len 1500})
+```
+
+This will produce an exception. Sometimes though, getting the first `n` items of
```

Comment:
> This will produce an exception.

How about: "This sequence will be replaced by an exception placeholder" or something along those lines? I was confused, reading that Realize would throw an exception, which it should never do.